Dijkstra Algorithm/README.md

```diff
@@ -0,0 +1,45 @@
+# Dijkstra-algorithm
+
+## About this repository
+This repository contains to playgrounds:
+* To understand how does this algorithm works, I have created **VisualizedDijkstra.playground.** It works in auto and interactive modes. Moreover there are play/pause/stop buttons.
+* If you need only realization of the algorithm without visualization then run **Dijkstra.playground.** It contains necessary classes and couple functions to create random graph for algorithm testing.
+
+## Demo video
+Click the link: [YouTube](https://youtu.be/PPESI7et0cQ)
+
+## Screenshots
+<img src="Images/screenshot1.jpg" height="400" />
+
+## Dijkstra's algorithm explanation
+
+<img src="Images/Dijkstra_Animation.gif" height="250" />
```

Comment:
This diagram needs more context for the reader. 

Consider that the reader is new and first opens this article. That person wouldn't and shouldn't know what this diagram is. You'll want to place this diagram in between an explanation to give it context.